apps/admin/integration-testing/.env

```diff
@@ -0,0 +1 @@
+REACT_APP_VX_SKIP_CAST_VOTE_RECORDS_AUTHENTICATION=TRUE
```

Comment:
> The main challenge I encountered was CVR authentication. The import failed with OpenSSL error "Verification Failed." I'm not sure if the instance would need to be set up with certain certificates - @arsalansufi would that be an easy fix?

Hmm interesting - integration tests _should_ be using our dev certs, i.e. the certs committed to [libs/auth/certs/dev](https://github.com/votingworks/vxsuite/tree/main/libs/auth/certs/dev) and pointed to via these functions:

https://github.com/votingworks/vxsuite/blob/c36e82ba5d531624b53e7bb781c276313c9a3391/libs/auth/src/config.ts#L24-L28

https://github.com/votingworks/vxsuite/blob/c36e82ba5d531624b53e7bb781c276313c9a3391/libs/auth/src/config.ts#L30-L63

And I've signed our CVR fixtures using those same dev keys and certs:

https://github.com/votingworks/vxsuite/blob/c36e82ba5d531624b53e7bb781c276313c9a3391/libs/cvr-fixture-generator/src/cli/generate/main.ts#L281-L282

Seems like something just isn't matching up. Skipping CVR auth is a totally fine workaround for now, and if you want to open a follow-up issue, I'll take a look when I get the chance!